libraries/AC_Avoidance/AP_OADatabase.h

```diff
@@ -31,20 +31,27 @@ class AP_OADatabase {
         OA_DbItemImportance importance;
     };
 
+    struct OA_DBItem_Tracked{
```

Comment:
let's put a comment above this structure saying what type of objects/items are we using this for.